frontend/lib/identification/card_detail_view/card_detail_view.dart

```diff
@@ -144,7 +145,9 @@ enum CardStatus {
   // The card was verified lately by the server and it responded that the card is invalid.
   invalid,
   // In any other case, we assume the card is valid and show the dynamic QR code
-  valid;
+  valid,
+  // The validity period didn't start yet
+  stillInvalid;
```

Comment:
how about `notYetValid`? stillInvalid can also mean that the card was invalid at some point and is still invalid.